app/assets/javascripts/visualizations/highvis/photos.coffee

```diff
@@ -43,63 +43,45 @@ $ ->
       # Gets called when the controls are clicked and at start
       update: ->
         # Clear the old canvas
-        ($ '#' + @canvas).html('')
-
-        ($ '#' + @canvas).append '<div id="polaroid"></div>'
-
-        i = 0
-        imgWidth = 200
-        defMargins = 20
-        excess = 0
-        for ds of data.metadata
-          numPhotos += data.metadata[ds].photos.length
-        divWidth = ($ '#polaroid').width()
-        if((imgWidth + defMargins) * numPhotos + defMargins < divWidth )
-          defMargins = 20
-        else
-          imgsPerLine = Math.floor(divWidth / (imgWidth + defMargins))
-          if imgsPerLine * (imgWidth + defMargins) + defMargins >= divWidth
-            imgsPerLine -= 1
-          excess = divWidth - (imgsPerLine * (defMargins + imgWidth) + 20)
-        ($ '#polaroid').css( 'margin-left': "#{(defMargins / 2) + (excess / 2)}px",
-          'margin-right': "#{(defMargins / 2) + (excess / 2)}px",
-          'margin-top': "#{defMargins / 2}px", 'margin-bottom': "#{defMargins / 2}px")
-        for ds of data.metadata
-          if data.metadata[ds].photos.length > 0
-            for pic of data.metadata[ds].photos
-              tmp = data.metadata[ds].photos[pic]
-              dset = data.metadata[ds]
-              do(tmp, dset) ->
-                figure = """<div class='p_item'>
-                  <img id='pic_#{i}' src="#{tmp.tn_src}" class='caroucell'/> <br>
-                  <div class="caption_wrap"><span class="caption">Data Set: #{dset.name}(#{dset.dataset_id})
-                  </span></div> <br>
-                </div>"""
-                ($ "#polaroid").append figure
-                ($ '#pic_' + i).css( cursor: "pointer")
-                ($ '#pic_' + i).click ->
-                  ($ '#polaroid').append("""
-                    <div class="modal fade" id="target_img" tabindex='-1'>
-                      <div class="modal-dialog">
-                        <div class="modal-content">
-                          <div class="modal-header">
-                            <button type="button" class="close" data-dismiss="modal"
-                              aria-hidden="true"><i class="fa fa-times"></i> Close</button>
-                          </div>
-                          <div class="modal-body">
-                            <img src='#{tmp.src}' style='width:100%'/>
-                          </div>
-                        </div>
-                      </div>
-                    </div>
-                    """)
-
-                  ($ '#target_img').modal
-                    keyboard: true
-                  ($ '#target_img').on "hidden.bs.modal", ->
-                    ($ '#target_img').remove()
-              i++
-              ($ '.p_item').css('margin': "#{defMargins / 2}px")
+        canvas = '#' + @canvas
+        $(canvas).html('<br/>')
```

Comment:
two things to note here:
1. technically, the '<br/>' is html, so it should have ``"``
2. the <br/> should even be there - just clear the canvas cleanly with ``$(canvas).html('');``